src/commons/navigationBar/NavigationBar.tsx

```diff
@@ -385,12 +393,7 @@ export const createDesktopNavlink: CreateNavlinkFunction = navbarEntry => (
     title={navbarEntry.text}
   >
     <Icon icon={navbarEntry.icon} />
-    <div
-      className={classNames(
-        'navbar-button-text',
```

Comment:
Is this `navbar-button-text` class no longer required?

Reply:
No, previously `navbar-button-text` was needed to fix spacing issues caused by children of `NavigationBar__link`. Since we removed the latter class from the parent component, the spacing is automatically fixed (and in fact, broken if we keep this class).